shared/src/constants/externalAddresses.json

```diff
@@ -82,6 +83,7 @@
       "wsteth": "0x7f39C581F595B53c5cb19bD0b3f8dA6c935E2Ca0",
       "ldo": "0x5A98FcBEA516Cf06857215779Fd812CA3beF1B32",
       "aave": "0x7Fc66500c84A76Ad7e9c93437bFc5Ac33E2DDaE9",
+      "uni": "0x1f9840a85d5af5bf1d1762f925bdaddc4201f984",
       "sushiLp": "0x9a13867048e01c663ce8ce2fe0cdae69ff9f35e3",
       "digg": "0x798d1be841a82a273720ce31c822c61a67a601c3",
       "rETH-THETA-v1": "0x0FABaF48Bbf864a3947bdd0Ba9d764791a60467A",
```

Comment:
Do we need to add the `rUNI-THETA-v2` address below as well?

Reply:
dont believe it is used anywhere in the repo, will delete them later

Reply:
those addresses are found in v2.deployments.json